build.gradle

```diff
@@ -242,7 +242,7 @@ project('spring-rabbit') {
 
 		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
 
-		testCompile "commons-cli:commons-cli:$commonsCliVersion"
+//		testCompile "commons-cli:commons-cli:$commonsCliVersion"
 
```

Comment:
I wondered why do we need this dependency...

And looks like there is nothing on the matter.

Although I have a problem with the `MessageListenerContainerRetryIntegrationTests.testStatefulRetryWithNoMessageIds()`, but not sure that it is related... I might have just missed something about Retry.